docs/run/error-mitigation-explanation.ipynb

```diff
@@ -0,0 +1,284 @@
+    "\n",
+    "![Depiction of gate twirls](/images/run/error-mitigation-explanation/gate_twirls.png)\n",
+    "\n",
+    "Pauli twirling can be enabled by setting `enable_gates` to `True` in the [twirling options](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.TwirlingOptions). Other notable options include:\n",
+    "\n",
+    "- `num_randomizations`: The number of circuit instances to draw from the ensemble of twirled circuits.\n",
+    "- `shots_per_randomization`: The number of shots to sample from each circuit instance.\n",
+    "\n",
+    "The following code cell shows how to enable Pauli twirling and set these options for the estimator. None of these options are required to be set explicitly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "c3e12d44",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "estimator = Estimator(backend=backend)\n",
+    "estimator.options.twirling.enable_gates = True\n",
+    "estimator.options.twirling.num_randomizations = 32\n",
+    "estimator.options.twirling.shots_per_randomization = 100"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "ad9d9159-4277-4aec-bf26-da748e240f38",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## Twirled readout error extinction (TREX)\n",
+    "\n",
+    "[Twirled readout error extinction (TREX)](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.105.032620) mitigates the effect of measurement errors for the estimation of Pauli observable expectation values.\n",
+    " It is based on the notion of twirled measurements, which are accomplished by randomly substituting measurement gates by a sequence of (1) a Pauli X gate, (2) a measurement, and (3) classical bit flip. Just like in standard gate twirling, this sequence is equivalent to a plain measurement in the absence of noise, as depicted in the following diagram:\n",
+    "\n",
+    "![Depiction of measurement twirling](/images/run/error-mitigation-explanation/measurement_twirling.png)\n",
+    "\n",
+    "In the presence of readout error, measurement twirling has the effect of diagonalizing the readout-error transfer matrix, making it easy to invert. Estimating the readout-error transfer matrix requires executing additional calibration circuits, which introduces a small overhead.\n",
+    "\n",
+    "TREX can be enabled by setting `measure_mitigation` to `True` in version 2 of the [Qiskit Runtime resilience options](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ResilienceOptionsV2). Options for measurement twirling are described [here](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.MeasureNoiseLearningOptions). As with gate twirling, you can set the number of circuit randomizations and the number of shots per randomization.\n",
```

Comment:
1. The class is called `ResilienceOptionsV2` because it's for V2 Estimator. Saying it's "version 2 of resilience options" might make people think you can use v1 options with v2 primitives. 
2. Since the link points to `MeasureNoiseLearningOptions` (same as the code), I assume that's what you're referring to. 


```suggestion
    "TREX can be enabled by setting `measure_mitigation` to `True` in the [Qiskit Runtime resilience options](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ResilienceOptionsV2) for V2 Estimator. Options for measurement noise learning are described [here](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.MeasureNoiseLearningOptions). As with gate twirling, you can set the number of circuit randomizations and the number of shots per randomization.\n",
```